scene/gui/color_picker.cpp

```diff
@@ -608,13 +608,14 @@ void ColorPicker::_update_presets() {
 
 #ifdef TOOLS_ENABLED
 	if (editor_settings) {
-		// Only load preset buttons when the only child is the add-preset button.
-		if (preset_container->get_child_count() == 1) {
-			for (int i = 0; i < preset_cache.size(); i++) {
-				_add_preset_button(preset_size, preset_cache[i]);
-			}
-			_notification(NOTIFICATION_VISIBILITY_CHANGED);
+		// Always update according to cache
```

Comment:
```suggestion
		// Always update according to cache.
```

Reply:
Although the comment is not really necessary now. You could remove it.

Reply:
You're right, I thought about removing then ended up opting to use this opportunity to clarify why the first button isn't removed as well :)